src/app/pages/ArticlePage/recommendationsExperiment/index.ts

```diff
@@ -48,7 +49,10 @@ export const transformRecsData = ({
   pageBlocks,
   variation,
 }: TransformRecsDataProps) => {
-  if (!variation) return wsojRecs;
+  // Optimizely will return null initially and then 'off' for users who are not in the experiment
+  if (!variation || variation === 'off') return wsojRecs;
```

Comment:
As the comment suggests, I've added an explicit `wsoj_off` variation to cater for our "off" experiment use case.

`null` and `off` here will be for users who are not and won't ever be in the experiment, so that they still get recommendations displayed

The explicit `wsoj_off` variation is for users in the experiment who will get shown no recommendations.